docs/content/Dialog.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 title: Dialog
-status: Alpha
+status: Deprecated
 ---
 
-import data from '../../src/Dialog.docs.json'
+import data from '../../src/deprecated/Dialog/Dialog.docs.json'
```

Comment:
I am hesitant to move the current Dialog to deprecated folder. Because

- First, I think we should issue a deprecation notice in a minor release to let folks know that this is what we are planning to do. And then in the major release (v36 might be too soon to do that?) we can move the Dialog to the deprecated folder. 
- Secondly, as you said, we don't have a Dialog in the main bundle. I think we should have because it might feel like we are deprecating the current one but we are still not ready to encourage adoption to the new dialog. I am curious what you think and what others think as well.

Reply:
ok, I'm in favor of moving the "old" dialog back to the main bundle 👍